test/e2e/apps/deployment.go

```diff
@@ -134,6 +141,283 @@ var _ = SIGDescribe("Deployment", func() {
 	})
 	// TODO: add tests that cover deployment.Spec.MinReadySeconds once we solved clock-skew issues
 	// See https://github.com/kubernetes/kubernetes/issues/29229
+
+	ginkgo.It("should run the lifecycle of a Deployment", func() {
+		deploymentResource := schema.GroupVersionResource{Group: "apps", Version: "v1", Resource: "deployments"}
+		testNamespaceName := f.Namespace.Name
+		testDeploymentName := "test-deployment"
+		testDeploymentInitialImage := "nginx"
```

Comment:
@Riaankl @hh if this is going to be a conformance, then we should make sure we use the images from the ones we use for conformance. right?

Reply:
Update to use conformance approved images n ee29022ac8a